CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to Grafana's Shared Workflows Repository
+
+Welcome to the Shared Workflows repository! This repository contains reusable actions designed to streamline and automate common tasks across multiple departments and external organizations. We appreciate your interest in contributing to our shared efforts.
+
+## Types of Workflows
+
+### 1. Reusable Workflows
+Reusable workflows are located in the `.github/workflows/` directory. These workflows are primarily intended for internal use within this repository. They should be used for automating tasks specific to this repository's needs. Reusable workflows are also required if you intend to use the `on: workflow_call` event for workflow triggering. For more information on reusing workflows and using `on: workflow_call`, refer to the [GitHub documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
+### 2. Custom Actions
+Custom actions are defined in the `actions/` directory. These actions are designed for external contributors to add new functionalities and actions that can be used across various projects. If you have innovative ideas or new actions to contribute, the `actions/` directory is the place to do so. To learn more about creating custom actions, check out the [GitHub documentation](https://docs.github.com/en/actions/creating-actions/about-custom-actions).
+
+## Contribution Guidelines
+
+1. **Fork the Repository:** Start by forking the repository to your GitHub account.
+
+2. **Clone the Fork:** Clone your forked repository to your local machine.
+
+3. **Create a Branch:** Create a new branch for your contribution. Use a descriptive name related to the changes you're making.
+
+4. **Make Changes:** Make your desired changes to the codebase.
+
+5. **Commit Changes:** Commit your changes with clear and concise commit messages.
+
+6. **Push Changes:** Push your changes to your forked repository.
+
+7. **Submit a Pull Request:** Submit a pull request to the main repository. Ensure that your pull request is detailed and includes information about the changes made.
```

Comment:
I would add something about checking CI results here.

There's also a CLA requirement which people will be notified about if they need to sign it.

Reply:
Updated to include this. WDYT?

```
11. **Contributor License Agreement (CLA):** Before we can accept your pull request, you need to sign [Grafana's CLA](https://grafana.com/docs/grafana/latest/developers/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
```

Reply:
excellent